src/layouts/InterfaceLayout/components/InterfaceMobileMenu/InterfaceMobileMenu.config.js

```diff
@@ -0,0 +1,151 @@
+export default {
```

Comment:
can you merge the changes from https://github.com/MyEtherWallet/MyEtherWallet/pull/1330
this file shouldnt be here